### PR TITLE
added Text to QR code Feature

### PR DESCRIPTION
**Pull request by iliakhaburdzania**

### CODE REVIEW

File: Printer/Command.swift [hunk 1]
```diff
@@ -78,4 +78,28 @@ extension ESC_POSCommand {
     static func underline(mode: UInt8) -> ESC_POSCommand {
         return ESC_POSCommand([27, 45, mode])
     }
+
```
Comment (KevinGong2013) [inline Printer/Command.swift]: add space after comma.

**Comment by KevinGong2013** [comment]
Thanks @iliakhaburdzania. Will merge after conflicts resolved.

**Comment by KevinGong2013** [comment]
#5

File: Printer/Command.swift [hunk 2]
```diff
@@ -78,4 +78,28 @@ extension ESC_POSCommand {
...
+        let pl = s % 256
+        let ph = s / 256
+        
+        return ESC_POSCommand([29,40,107,UInt8(pl),UInt8(ph),49,80,48])
+    }
+    
+    
+    static func QRPrint() -> ESC_POSCommand {
+        return ESC_POSCommand([29,40,107,3,0,49,81,48])
+    }
```
Comment (KevinGong2013) [inline Printer/Command.swift:104]: add space after comma.

Comment (iliakhaburdzania) [reply Printer/Command.swift:104]: Done ✅

**Pull request merged**